common/constants.go

```diff
@@ -63,6 +63,7 @@ const (
 	EncodingTypeGob      EncodingType = "gob"
 	EncodingTypeUnknown  EncodingType = "unknow"
 	EncodingTypeEmpty    EncodingType = ""
+	EncodingTypeProto    EncodingType = "proto3"
```

Comment:
Shouldn't it be `EncodingTypeProto3`?

Reply:
Figured we wont have proto2 for quite some time considering proto3 wire format is backwards compatible :) But will add to name